README.md

```diff
@@ -1,99 +1,63 @@
-Poseidon is Firmament's (http://www.firmament.io) integration with
-Kubernetes.
+# Introduction
+The Poseidon/Firmament scheduler incubation project is to bring integration of Firmament Scheduler (OSDI paper) in Kubernetes.
```

Comment:
Should there be a link to the paper here?

Reply:
Yes, we definitely should.
@shivramsrivastava, The original document I gave it to you had link for OSDI paper. Can we please  enable the hyperlink for "OSDI paper".